lib/views/home/home_view.dart

```diff
@@ -21,151 +21,95 @@ class HomeView extends ConsumerStatefulWidget {
-                        ),
-                        Column(
-                          children: [
-                            _getAnnouncementBanner(data),
-                            height24,
-                            FilterWidget(
-                              chips: data.chips,
-                            ),
-                            height32,
-                            _cardListWidget(data),
-                            height32,
-                            height32,
-                            height32,
-                          ],
-                        ),
-                      ],
+      body: SafeArea(
+        bottom: false,
+        child: Container(
+          child: Column(
+            mainAxisSize: MainAxisSize.min,
+            children: [
+              Expanded(
+                child: RefreshIndicator(
+                  onRefresh: _onRefresh,
+                  child: SingleChildScrollView(
+                    physics: AlwaysScrollableScrollPhysics(
+                      parent: BouncingScrollPhysics(),
+                    ),
+                    child: Container(
+                      color: ColorConstants.ebony,
+                      child: Column(
+                        children: [
+                          HeaderAndAnnouncementWidget(),
+                          height16,
+                          ShortcutsWidget(),
+                          height24,
+                          QuoteWidget(),
+                          height24,
+                          TilesWidget(),
+                          SizedBox(
+                            height: 120,
+                          ),
+                        ],
+                      ),
                     ),
                   ),
                 ),
               ),
-            ),
-          ],
+            ],
+          ),
         ),
       ),
     );
   }
 
+  Future<void> _onRefresh() async {
+    ref.invalidate(homeProvider);
+    await ref.read(homeProvider.future);
+    ref.invalidate(remoteStatsProvider);
+    await ref.read(remoteStatsProvider.future);
+    ref.invalidate(fetchShortcutsProvider);
+    await ref.read(fetchShortcutsProvider.future);
+    ref.invalidate(fetchQuoteProvider);
+    await ref.read(fetchQuoteProvider.future);
+    ref.invalidate(fetchHomeHeaderProvider);
+    await ref.read(fetchHomeHeaderProvider.future);
+  }
+
+  void onPointerUp(_) async {
+    if (await Vibration.hasVibrator() ?? false) {
+      if ((await Vibration.hasAmplitudeControl()) ?? false) {
+        unawaited(Vibration.vibrate(amplitude: 50, duration: 5));
+      } else {
+        unawaited(Vibration.vibrate(duration: 5));
+      }
+    }
+  }
+
+  void onPointerDown(_) async {
+    if (await Vibration.hasVibrator() ?? false) {
+      if ((await Vibration.hasAmplitudeControl()) ?? false) {
+        unawaited(Vibration.vibrate(amplitude: 20, duration: 30));
+      } else {
+        unawaited(Vibration.vibrate(duration: 30));
+      }
+    }
+  }
+
```

Comment:
Let's remove this vibration thing. I spoke to @scouech about it. It's really different on all devices, some devices have a really weak vibration and some a really strong one so it's not a great UX.

Reply:
Removed